packages/wdio-sync/package.json

```diff
@@ -17,6 +17,7 @@
     "test:eslint": "eslint src test",
     "test:unit": "jest"
   },
+  "types": "./webdriverio-sync.d.ts",
```

Comment:
Can we just name this `webdriverio.d.ts`? By being within the `@wdio/sync` package it should be hint enough that these are the sync types.

Reply:
There are files named webdriverio.d.ts in both webdriverio and @wdio/sync.

Should I rename them to webdriverio-core.d.ts? Will that work?

Reply:
> There are files named webdriverio.d.ts in both webdriverio and @wdio/sync.

Does this matter? Typescript is only looking what "types" is referencing too, no? So they can both be the same.

Reply:
Not really, but we have to keep two files physically.
I'll rename webdriverio to webdriverio-core, and webdriverio-sync and async to webdriverio.

Reply:
@christian-bromann updated